schedule/publiccloud/staging_images/qem_publiccloud_img_proof.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: qem_publiccloud_img_proof
+description: |
+  img_proof publiccloud test
+schedule:
+  - boot/boot_to_desktop
+  - publiccloud/ssh_interactive_init
```

Comment:
why img_proof test needs `ssh_interactive_init` ?

Reply:
`ssh_interactive_init` creates the VM under test. You might confuse it with `ssh_interactive_start`.